Misc/NEWS.d/next/Library/2018-12-09-21-35-49.bpo-20239.V4mWBL.rst

```diff
@@ -0,0 +1,2 @@
+Allow repeated assignment deletion of ``unittest.mock.Mock`` attributes.
```

Comment:
You might want to link to the class here.

Reply:
Done in 0c3dde0